File: exercice3_1.c
#include<stdio.h>
#include<stdlib.h>

int main(int argc, char *arg[])
{
    if(argc != 3)
    {
        printf("Usage : %s argument1 argument2\n",arg[0]);
        return(0);
    }
    int entier1=atoi(arg[1]); //converti un chaine de caractere en entier
    int entier2=atoi(arg[2]);
    int resultat;
    resultat=entier1+entier2;
    printf("le resultat des sommes:\n%d + %d = %d\n",entier1,entier2,resultat);

    return(0);
}

File: exercice3_1.py
import sys

def main():
    if len(sys.argv) != 3:
        print(f"Usage : {sys.argv[0]} argument1 argument2")
        return

    entier1 = int(sys.argv[1])
    entier2 = int(sys.argv[2])

    resultat = entier1 + entier2

    print("le resultat des sommes:")
    print(f"{entier1} + {entier2} = {resultat}")

if __name__=="__main__":
  main()